controllers/chats.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * chat.js
+ * Controllers for interacting with chat documents in the database.
+ */
+
+const db = require('../db/connect');
+
+const users = require('./users');
+
+const Chat = db.models.Chat;
+
+// const UUID = require('uuid-1345');
+// require('mongoose-uuid2')(mongoose);
+
+/**
+* addChatFeedback
+* Take a chat object containing all the data listed in the chat schema
+* also includes a feedback object
+* append that that feedback object to the chat feedback attribute which is a list
+* verify that ChatSchema.feedback is of length >= 2
+* takes a feedback object
+*/
+
+function logFeedback(feedback) {
+    console.log("Chat: adding feedback");
```

Comment:
can you make this logging a little bit clearer? i can't tell 100% what is happening